index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio 👨‍💻 </title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    
+    <header>
+        <h2>My logo</h2>
+       <nav>
+           <button type="button" class ="btn">&#9776;</button>
+       </nav>
+    </header>
+
+    <section class="headline-section">
+
+                <h1>I’m Abdifatah Yassen<br>
+                    Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you<br> build a product , feature or website Look<br>
+                         through some of my work and <br>experience! If you like what you see and<br> have a project you need
+                        coded, don’t <br>hestiate to contact me.</p>
+                        <p id="text">LET’S CONNECT</p>
```

Comment:
- [ ] Kindly add a bit more spacing on the top and bottom of the Let's Connect text. Currently, it's cluttered and that is not a good experience and is not as close as the Figma template given. Please work on it 👍

**See Your Design**
![image](https://user-images.githubusercontent.com/50941074/170673114-fa9b5d2d-b14b-413c-bd7e-2d0f6293395e.png)

Reply:
Done